spec/comparators/hashed_configuration_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../spec_helper'
+
+describe HashedConfiguration do
```

Comment:
Yeah, but kind of remove the necesity of branching each comparator test between a string and a hash.